docs/src/cli/delegate-stake.md

```diff
@@ -0,0 +1,141 @@
+# Earn Staking Rewards
+
+After you have [received SOL](transfer-tokens.md), you might consider putting
+it to use by delegating *stake* to a validator. Stake is what we call tokens
+in a *stake account*. Solana weights validator votes by the amount of stake
+delegated to them, which gives those validators more influence in determining
+then next valid block of transactions in the blockchain. Solana then generates
+new SOL periodically to reward stakers and validators. You earn more rewards
+the more stake you delegate.
+
+## Create a Stake Account
+
+To delegate stake, you will need to transfer some tokens into a stake account.
+Create a stake account with the `solana create-stake-account` command:
+
+```bash
+solana-keygen new --no-passphrase -o stake-account.json
+solana create-stake-account --from=<KEYPAIR> stake-account.json <AMOUNT> --stake-authority=<KEYPAIR> --withdraw-authority=<KEYPAIR>
+```
+
+`<AMOUNT>` tokens are transferred from the account at `<KEYPAIR>` to a new
+stake account at the public key of stake-account.json.  Record the account
+public key, as it will be used later to perform actions on the stake account.
+The stake-account.json file can then be discarded. To authorize additional
+actions, you will use `<KEYPAIR>`, not the keypair at stake-account.json.
```

Comment:
```suggestion
public key, as it will be used later as the `<STAKE_ACCOUNT_ADDRESS>`
when performing actions on the stake account.  The stake-account.json file
can then be discarded. To authorize additional actions, you will use
`<KEYPAIR>`, not the keypair at stake-account.json.
```

Reply:
But the call to `create-stake-account` was above

Reply:
@t-nelson, rewrote it

Reply:
Ah! Looks like my attempt to uphold line wrap broken the fine grained diff highlighting.  My goal was to link the throwaway keypair to the `<STAKE_ACCOUNT_ADDRESS>`.  I don't believe we make a smooth transition there